### ISSUE TITLE
After upgrading classgraph I can no longer find classes with own classpath with urls starting with jar:/// and jar:file:

### ISSUE THREAD

**Issue by alexsuter** [classgraph/classgraph#766]
After upgrading classgraph from 4.8.149 to 4.8.157 I can no longer find classes with own classpath with urls starting with jar:/// and jar:file:

e.g.:
- `jar:////home/any/resources/ProjectWithAnnotations.iar!classes/`
- `jar:file:/tmp/junit9971740069830709806/myProject.iar.zip!classes/`

but others still work e.g.:
- `file:/home/any/resources/ProjectWithAnnotations.iar!classes/`

I tired to enable #verbose, but I don't get any logs...

**Comment by lukehutch** [comment]
You get no logs at all with #verbose?

Any chance you can provide a minimal test case?

**Comment by alexsuter** [comment]
I made a repo for this issue:
https://github.com/alexsuter/classgraph-issue766

It seems that the change in behavior has been introduced with classgraph 4.8.151.

In this dedicated repository, I got also a verbose log output. I guess I've some strange logger configuration in my other repository.

```
May 05, 2023 8:47:10 AM nonapi.io.github.classgraph.utils.LogNode flush
INFO: 2023-05-05T08:47:10.015+0200	ClassGraph	ClassGraph version 4.8.150
2023-05-05T08:47:10.017+0200	ClassGraph	Operating system: Linux 5.19.0-41-generic amd64
2023-05-05T08:47:10.017+0200	ClassGraph	Java version: 17.0.6 / 17.0.6+10 (Eclipse Adoptium)
2023-05-05T08:47:10.017+0200	ClassGraph	Java home: /home/alex/eclipse/eclipse-rcp-2023-03-R-linux-gtk-x86_64/plugins/org.eclipse.justj.openjdk.hotspot.jre.full.linux.x86_64_17.0.6.v20230204-1729/jre
2023-05-05T08:47:10.024+0200	ClassGraph	ScanSpec:
2023-05-05T08:47:10.024+0200	ClassGraph	-- packageAcceptReject: 
2023-05-05T08:47:10.024+0200	ClassGraph	-- packagePrefixAcceptReject: 
2023-05-05T08:47:10.024+0200	ClassGraph	-- pathAcceptReject: 
2023-05-05T08:47:10.024+0200	ClassGraph	-- pathPrefixAcceptReject: 
2023-05-05T08:47:10.024+0200	ClassGraph	-- classAcceptReject: 
2023-05-05T08:47:10.024+0200	ClassGraph	-- classfilePathAcceptReject: 
2023-05-05T08:47:10.024+0200	ClassGraph	-- classPackageAcceptReject: 
2023-05-05T08:47:10.024+0200	ClassGraph	-- classPackagePathAcceptReject: 
2023-05-05T08:47:10.024+0200	ClassGraph	-- moduleAcceptReject: 
2023-05-05T08:47:10.024+0200	ClassGraph	-- jarAcceptReject: 
2023-05-05T08:47:10.024+0200	ClassGraph	-- classpathElementResourcePathAcceptReject: 
2023-05-05T08:47:10.024+0200	ClassGraph	-- libOrExtJarAcceptReject: 
2023-05-05T08:47:10.025+0200	ClassGraph	-- scanJars: true
2023-05-05T08:47:10.025+0200	ClassGraph	-- scanNestedJars: false
2023-05-05T08:47:10.025+0200	ClassGraph	-- scanDirs: true
2023-05-05T08:47:10.025+0200	ClassGraph	-- scanModules: true
2023-05-05T08:47:10.025+0200	ClassGraph	-- enableClassInfo: true
2023-05-05T08:47:10.025+0200	ClassGraph	-- enableFieldInfo: false
2023-05-05T08:47:10.025+0200	ClassGraph	-- enableMethodInfo: false
2023-05-05T08:47:10.025+0200	ClassGraph	-- enableAnnotationInfo: true
2023-05-05T08:47:10.025+0200	ClassGraph	-- enableStaticFinalFieldConstantInitializerValues: false
2023-05-05T08:47:10.025+0200	ClassGraph	-- enableInterClassDependencies: false
2023-05-05T08:47:10.025+0200	ClassGraph	-- enableExternalClasses: false
2023-05-05T08:47:10.025+0200	ClassGraph	-- enableSystemJarsAndModules: false
2023-05-05T08:47:10.025+0200	ClassGraph	-- ignoreClassVisibility: false
2023-05-05T08:47:10.025+0200	ClassGraph	-- ignoreFieldVisibility: false
2023-05-05T08:47:10.025+0200	ClassGraph	-- ignoreMethodVisibility: false
2023-05-05T08:47:10.025+0200	ClassGraph	-- disableRuntimeInvisibleAnnotations: false
2023-05-05T08:47:10.026+0200	ClassGraph	-- extendScanningUpwardsToExternalClasses: true
2023-05-05T08:47:10.026+0200	ClassGraph	-- allowedURLSchemes: null
2023-05-05T08:47:10.026+0200	ClassGraph	-- addedClassLoaders: null
2023-05-05T08:47:10.026+0200	ClassGraph	-- overrideClassLoaders: null
2023-05-05T08:47:10.026+0200	ClassGraph	-- addedModuleLayers: null
2023-05-05T08:47:10.026+0200	ClassGraph	-- overrideModuleLayers: null
2023-05-05T08:47:10.026+0200	ClassGraph	-- overrideClasspath: [jar:////home/alex/eclipse-workspace/core/classgraph-issue766/target/classes/com/github/alexsuter/classgraph/issue766/ProjectWithAnnotations.iar]
2023-05-05T08:47:10.026+0200	ClassGraph	-- classpathElementFilters: null
2023-05-05T08:47:10.026+0200	ClassGraph	-- initializeLoadedClasses: false
2023-05-05T08:47:10.026+0200	ClassGraph	-- removeTemporaryFilesAfterScan: false
2023-05-05T08:47:10.026+0200	ClassGraph	-- ignoreParentClassLoaders: false
2023-05-05T08:47:10.026+0200	ClassGraph	-- ignoreParentModuleLayers: false
2023-05-05T08:47:10.026+0200	ClassGraph	-- modulePathInfo: 
2023-05-05T08:47:10.026+0200	ClassGraph	-- maxBufferedJarRAMSize: 67108864
2023-05-05T08:47:10.026+0200	ClassGraph	-- enableMemoryMapping: false
2023-05-05T08:47:10.027+0200	ClassGraph	Number of worker threads: 24
2023-05-05T08:47:10.031+0200	ClassGraph	Finding classpath
2023-05-05T08:47:10.031+0200	ClassGraph	-- Finding classpath and modules
2023-05-05T08:47:10.039+0200	ClassGraph	---- Overriding classpath with: [jar:////home/alex/eclipse-workspace/core/classgraph-issue766/target/classes/com/github/alexsuter/classgraph/issue766/ProjectWithAnnotations.iar]
2023-05-05T08:47:10.039+0200	ClassGraph	------ Found classpath element: /home/alex/eclipse-workspace/core/classgraph-issue766/target/classes/com/github/alexsuter/classgraph/issue766/ProjectWithAnnotations.iar
2023-05-05T08:47:10.040+0200	ClassGraph	------ WARNING: when the classpath is overridden, there is no guarantee that the classes found by classpath scanning will be the same as the classes loaded by the context classloader
2023-05-05T08:47:10.044+0200	ClassGraph	Opening classpath elements (took 0.019614 sec)
2023-05-05T08:47:10.054+0200	ClassGraph	-- Opening classpath element file:///home/alex/eclipse-workspace/core/classgraph-issue766/target/classes/com/github/alexsuter/classgraph/issue766/ProjectWithAnnotations.iar
2023-05-05T08:47:10.054+0200	ClassGraph	---- Opening jar: file:///home/alex/eclipse-workspace/core/classgraph-issue766/target/classes/com/github/alexsuter/classgraph/issue766/ProjectWithAnnotations.iar
2023-05-05T08:47:10.064+0200	ClassGraph	Finding nested classpath elements
2023-05-05T08:47:10.064+0200	ClassGraph	Final classpath element order:
2023-05-05T08:47:10.064+0200	ClassGraph	-- /home/alex/eclipse-workspace/core/classgraph-issue766/target/classes/com/github/alexsuter/classgraph/issue766/ProjectWithAnnotations.iar
2023-05-05T08:47:10.064+0200	ClassGraph	Scanning classpath elements (took 0.007440 sec)
2023-05-05T08:47:10.065+0200	ClassGraph	-- Scanning jarfile classpath element /home/alex/eclipse-workspace/core/classgraph-issue766/target/classes/com/github/alexsuter/classgraph/issue766/ProjectWithAnnotations.iar (took 0.005724 sec)
2023-05-05T08:47:10.066+0200	ClassGraph	---- Found resource within accepted package: .classpath
2023-05-05T08:47:10.066+0200	ClassGraph	---- Found resource within accepted package: .project
2023-05-05T08:47:10.066+0200	ClassGraph	---- Found resource within subpackage of accepted package: .settings/.jsdtscope
2023-05-05T08:47:10.066+0200	ClassGraph	---- Found resource within subpackage of accepted package: .settings/ch.ivyteam.ivy.designer.prefs
2023-05-05T08:47:10.066+0200	ClassGraph	---- Found resource within subpackage of accepted package: .settings/org.eclipse.jdt.core.prefs
2023-05-05T08:47:10.066+0200	ClassGraph	---- Found resource within subpackage of accepted package: .settings/org.eclipse.wst.common.component
2023-05-05T08:47:10.066+0200	ClassGraph	---- Found resource within subpackage of accepted package: .settings/org.eclipse.wst.common.project.facet.core.prefs.xml
2023-05-05T08:47:10.066+0200	ClassGraph	---- Found resource within subpackage of accepted package: .settings/org.eclipse.wst.common.project.facet.core.xml
2023-05-05T08:47:10.066+0200	ClassGraph	---- Found resource within subpackage of accepted package: .settings/org.eclipse.wst.css.core.prefs
2023-05-05T08:47:10.066+0200	ClassGraph	---- Found resource within subpackage of accepted package: .settings/org.eclipse.wst.jsdt.ui.superType.container
2023-05-05T08:47:10.066+0200	ClassGraph	---- Found resource within subpackage of accepted package: .settings/org.eclipse.wst.jsdt.ui.superType.name
2023-05-05T08:47:10.066+0200	ClassGraph	---- Found resource within accepted package: .ulcproperties
2023-05-05T08:47:10.066+0200	ClassGraph	---- Found resource within subpackage of accepted package: META-INF/persistence.xml ; full path: classes/META-INF/persistence.xml
2023-05-05T08:47:10.066+0200	ClassGraph	---- Found classfile within subpackage of accepted package: ch/ivyteam/test/MyManagedBean.class ; full path: classes/ch/ivyteam/test/MyManagedBean.class
2023-05-05T08:47:10.073+0200	ClassGraph	------ Parsing classfile (took 0.014917 sec)
2023-05-05T08:47:10.086+0200	ClassGraph	-------- Found class ch.ivyteam.test.MyManagedBean
2023-05-05T08:47:10.086+0200	ClassGraph	---------- Superclass: java.lang.Object
2023-05-05T08:47:10.088+0200	ClassGraph	---------- Class annotations: @javax.annotation.ManagedBean
2023-05-05T08:47:10.088+0200	ClassGraph	---------- External class annotation javax.annotation.ManagedBean was not found in non-rejected packages -- cannot extend scanning to this class
2023-05-05T08:47:10.066+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Layouts/1Axonivy/1485F67ACAC259D7.jsp
2023-05-05T08:47:10.066+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Layouts/1Axonivy/co.meta
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Layouts/1PanelWithHeaderFooter/1179B65C31F3A109.jsp
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Layouts/1PanelWithHeaderFooter/co.meta
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Layouts/1centeredPanel/10EC2DD258C49EAF.jsp
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Layouts/1centeredPanel/co.meta
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Layouts/co.meta
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Project/Banner/1178345D699AF3CE.png
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Project/Banner/co.meta
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Project/FooterText/1178C05004C41EBD.data
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Project/FooterText/co.meta
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Project/HeaderText/1178C05685DDA3D7.data
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Project/HeaderText/co.meta
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Project/co.meta
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Styles/Axonivy/148600F8CD2D3747.css
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Styles/Axonivy/co.meta
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Styles/Blue/10EC2705822F2C1A.css
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Styles/Blue/co.meta
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Styles/Classic/10EC278AA7BE62DA.css
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Styles/Classic/co.meta
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Styles/Cool/10EC278D4F2834D5.css
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Styles/Cool/co.meta
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Styles/Ivy/10EC2791ECD64576.css
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Styles/Ivy/co.meta
2023-05-05T08:47:10.067+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/Styles/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/cms.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/system/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/system/validation/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/system/validation/mandatoryerrormessage/12655B706C5E443E.data
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: cms/system/validation/mandatoryerrormessage/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/BooleanFormat/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RButton/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RCheckBox/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RCheckBoxMenuItem/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RComboBox/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RComboBoxCellWidget/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RDatePicker/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RFiller/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RHtmlPane/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RHyperlink/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RLabel/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RList/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RLookupTextField/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RMenuItem/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RPasswordField/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RProgressBar/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RRadioButton/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RRadioButtonMenuItem/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RScrollBar/co.meta
2023-05-05T08:47:10.068+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RSlider/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RTable/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RTableCell/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RTableTree/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RTextArea/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RTextField/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RTextFieldCellWidget/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RToggleButton/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/RTree/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ComponentStyle/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ContainerStyle/RBorderLayoutPane/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ContainerStyle/RBoxLayoutPane/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ContainerStyle/RBoxPane/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ContainerStyle/RCardPane/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ContainerStyle/RCollapsiblePane/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ContainerStyle/RFlowLayoutPane/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ContainerStyle/RGridBagLayoutPane/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ContainerStyle/RGridLayoutPane/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ContainerStyle/RScrollPane/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ContainerStyle/RSplitPane/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ContainerStyle/RTabbedPane/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ContainerStyle/RTaskPane/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ContainerStyle/RTaskPaneContainer/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ContainerStyle/RToolBar/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/ContainerStyle/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/DateFormat/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/DateTimeFormat/co.meta
2023-05-05T08:47:10.069+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/DateTimeWidgetValidation/co.meta
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/DateWidgetValidation/co.meta
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/Environments/co.meta
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/GlobalVariables/co.meta
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/NumberFormat/co.meta
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/NumberWidgetValidation/co.meta
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/PercentWidgetValidation/co.meta
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/RegexWidgetValidation/co.meta
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/RichDialogWindowTypes/co.meta
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/SmartTable/co.meta
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/StringFormat/co.meta
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/TimeFormat/co.meta
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/TimeWidgetValidation/co.meta
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/cms.meta
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: config/co.meta
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within accepted package: databases.dbConfig
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: dataclasses/project/with/annotations/Data.ivyClass
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: lib/nested.jar
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within accepted package: overrides.overrideconfig
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within accepted package: pom.xml
2023-05-05T08:47:10.066+0200	ClassGraph	---- Found classfile within subpackage of accepted package: project/with/annotations/Data.class ; full path: classes/project/with/annotations/Data.class
2023-05-05T08:47:10.073+0200	ClassGraph	------ Parsing classfile (took 0.007751 sec)
2023-05-05T08:47:10.081+0200	ClassGraph	-------- Found class project.with.annotations.Data
2023-05-05T08:47:10.081+0200	ClassGraph	---------- Superclass: ch.ivyteam.ivy.scripting.objects.CompositeObject
2023-05-05T08:47:10.081+0200	ClassGraph	---------- External superclass ch.ivyteam.ivy.scripting.objects.CompositeObject was not found in non-rejected packages -- cannot extend scanning to this class
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within accepted package: restClients.restConfig
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within accepted package: roles.roleconfig
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: src/META-INF/persistence.xml
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: src/ch/ivyteam/test/MyManagedBean.java
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: src_dataClasses/project/with/annotations/Data.java
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within accepted package: testusers.userconfig
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: webContent/META-INF/MANIFEST.MF
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within subpackage of accepted package: webContent/WEB-INF/faces-config.xml
2023-05-05T08:47:10.070+0200	ClassGraph	---- Found resource within accepted package: webservice.wsconfig
2023-05-05T08:47:10.072+0200	ClassGraph	Masking classfiles (took 0.000045 sec)
2023-05-05T08:47:10.073+0200	ClassGraph	Scanning classfiles (took 0.016424 sec)
2023-05-05T08:47:10.090+0200	ClassGraph	Linking related classfiles (took 0.005557 sec)
2023-05-05T08:47:10.098+0200	ClassGraph	Total time: 0.053 sec
```

**Comment by lukehutch** [comment]
I'll take a look at your repo, thanks for putting that together.

**Comment by lukehutch** [comment]
Should be fixed in 4.8.158. Thanks for the report!

**Comment by alexsuter** [comment]
Thanks!
